src/objects/utils/views.py

```diff
@@ -23,3 +28,16 @@ def server_error(request, template_name=ERROR_500_TEMPLATE_NAME):
         )
     context = {"request": request}
     return http.HttpResponseServerError(template.render(context))
+
+
+def exception_handler(exc, context):
+    response = drf_exception_handler(exc, context)
+
+    # provide user-friendly response if data_icontains was used but DB couldn't process it
+    if not response and isinstance(exc, DatabaseError) and "jsonpath" in exc.args[0]:
+        data = {
+            "detail": "Objects DB doesn't support 'data_icontains'. Please, update it to Postgres 12."
```

Comment:
This is an error to the user, so the update to Postgres 12 message doesn't make much sense.

```suggestion
            "detail": "This search operation is not supported by the underlying data store."
```

I'm not sure if this 500 is then picked up by Sentry. If not, we should add `logger.exception`.

Reply:
`logger.exception` is added